backend/tox.ini

```diff
@@ -9,4 +9,3 @@ deps = flake8
        isort
 commands = flake8 --show-source app.py
            black --check app.py
-           isort --check-only app.py
```

Comment:
why not?

Reply:
It fails, but running `isort` on the fail, doesn't change anything - so I assume misconfiguration .. and I didn't wanna bother ;)

Reply:
weird